operators/src/plot/statistics.rs

```diff
@@ -118,7 +119,7 @@ impl PlotQueryProcessor for StatisticsQueryProcessor {
             queries.push(
                 call_on_generic_raster_processor!(raster_processor, processor => {
                     processor.query(query.into(), ctx).await?
-                             .map(move |r| r.map(|tile| (i, tile.convert::<f64>())))
+                             .map(move |r| r.map(|tile| (i, tile.convert_data_type())))
```

Comment:
Shouldn't we use the raster conversion adapter/processor here?

Reply:
it would require to clone the processor. i used the util method 
[326da66](https://github.com/geo-engine/geoengine/pull/462/commits/326da66b389bb16d725d8c755c295a41e8cf4370)